### PR TITLE
Jkmarx/dashboard search bug fix

### PR DESCRIPTION
**Pull request by jkmarx**
- Fix solr limit of 10
- Fix searching due to api call racing (added search button)
- Add extra blopping during wait time
- Fix invite ux

screen shot 2018-05-14 at 2 50 04 pm

searchgiphy

### CODE REVIEW

File: refinery/ui/source/js/commons/ctrls/group-member-add-modal-ctrl.js [hunk 1]
```diff
@@ -87,7 +87,6 @@
           vm.isLoading = false;
           generateAlertMessage('success', vm.form.email);
           groupDataService.update();
-          vm.form.email = '';
```
Comment (scottx611x) [inline refinery/ui/source/js/commons/ctrls/group-member-add-modal-ctrl.js]: Is this related to the PR?

File: refinery/ui/source/js/dashboard/controllers/data-sets-card-ctrl.js [hunk 2]
```diff
@@ -176,19 +179,32 @@
      * @param {string} query - user entered query for data set search
     **/
     function searchDataSets (query) {
+      // reset perm filter until we can search & check perms
+      vm.groupFilter = { selectedName: 'All' };
       if (query && query.length > 1) {
+        vm.loadingDataSets = true;
         var apiRequest = new DataSetSearchApi(query);
-        apiRequest().then(function (response) {
+        apiRequest(200).then(function (response) {
```
Comment (scottx611x) [inline refinery/ui/source/js/dashboard/controllers/data-sets-card-ctrl.js]: Was there a similar success response that wasn't `200` here before, or is this just to be explicit?

**Comment by jkmarx** [comment]
@scottx611x Waiting for the build to pass and will merge. I just needed to adjust the search bar css. Otherwise we should be ready to go.
